skiko/src/nativeJsMain/cpp/DirectContext.cc

```diff
@@ -25,12 +35,22 @@ SKIKO_EXPORT KNativePointer org_jetbrains_skia_DirectContext__1nMakeMetal
     sk_sp<GrDirectContext> instance = GrDirectContext::MakeMetal(backendContext);
     return reinterpret_cast<KNativePointer>(instance.release());
 }
-#endif
+#endif // SK_METAL
 
+#ifdef SK_DIRECT3D
```

Comment:
Maybe better always have a function implementation, but throw an exception similar to TODO() in such a case.